hazelcast/src/main/java/com/hazelcast/jet/impl/JetServiceBackend.java

```diff
@@ -200,28 +223,32 @@ public void shutDownJobs() {
     private void notifyMasterWeAreShuttingDown(CompletableFuture<Void> future) {
         Operation op = new NotifyMemberShutdownOperation();
         nodeEngine.getOperationService()
-                  .invokeOnTarget(JetServiceBackend.SERVICE_NAME, op, nodeEngine.getClusterService().getMasterAddress())
-                  .whenCompleteAsync((response, throwable) -> {
-                      // if there is an error and the node is still ACTIVE, try again. If the node isn't ACTIVE, log & ignore.
-                      NodeState nodeState = nodeEngine.getNode().getState();
-                      if (throwable != null && nodeState == NodeState.ACTIVE) {
-                          logger.warning("Failed to notify master member that this member is shutting down," +
-                                  " will retry in " + NOTIFY_MEMBER_SHUTDOWN_DELAY + " seconds", throwable);
-                          // recursive call
-                          nodeEngine.getExecutionService().schedule(
-                                  () -> notifyMasterWeAreShuttingDown(future), NOTIFY_MEMBER_SHUTDOWN_DELAY, SECONDS);
-                      } else {
-                          if (throwable != null) {
-                              logger.warning("Failed to notify master member that this member is shutting down," +
-                                      " but this member is " + nodeState + ", so not retrying", throwable);
-                          }
-                          future.complete(null);
-                      }
-                  });
+                .invokeOnTarget(JetServiceBackend.SERVICE_NAME, op, nodeEngine.getClusterService().getMasterAddress())
```

Comment:
I cannot really say if anything was edited. If reformat was by mistake, please revert it

Reply:
Ok. Reverted the formatting